src/main/java/uk/nhs/hee/tis/trainee/credentials/service/IssuanceService.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Optional;
 import java.util.UUID;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
```

Comment:
There are a couple of unused imports added here

Reply:
Ah, thanks - I wonder why sonar missed them. Fixed now